dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedTRSOpenApiIT.java

```diff
@@ -178,8 +178,10 @@ void testAggregatedMetricsPut() {
         assertNotNull(workflowVersion);
         assertEquals(2, workflowVersion.getMetricsByPlatform().size(), "Version should have metrics for 2 platforms");
 
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform1));
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform2));
+        String metricsGet = extendedGa4GhApi.aggregatedMetricsGet(workflowId, workflowVersionId);
```

Comment:
If it's helpful, I think you should be getting a Map of Partner to Metrics as in `getAggregatedMetrics` 
https://swagger.io/docs/specification/data-models/dictionaries/ has some info on how to define a `Map<Partner, Metrics>` as a return type rather than a `String`